Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -714,9 +714,10 @@ WarpX::PSATDScaleAverageFields (const amrex::Real scale_factor)
 #endif // WARPX_USE_FFT
 
 void
-WarpX::PushPSATD ()
+WarpX::PushPSATD (amrex::Real start_time)
```

Comment:
clang-tidy is complaining about the fact that we use `start_time` in the definition here, but `cur_time` in the declaration in WarpX.H. We need to use one single name to suppress that warning (which we treat as error), not sure which one you think is most appropriate.

Reply:
Fixed.